bjoern/request.c

```diff
@@ -97,22 +93,26 @@ void Request_parse(Request* request, const char* data, const size_t data_len)
     Py_DECREF(val); \
   } while(0)
 
-
-#define _set_header_from_http_header() \
+#define _set_or_append_header(k, val, len) \
```

Comment:
With this code getting more complex, I'd rather see it in a function than as a macro.

Reply:
i used macro because there was a macro before, and was thinking about inline functions. It was converted to a static function.